upup/models/cloudup/resources/addons/networking.flannel/v0.7.0.yaml

```diff
@@ -0,0 +1,89 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: flannel
+  namespace: kube-system
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kube-flannel-cfg
+  namespace: kube-system
+  labels:
+    k8s-app: flannel
+data:
+  cni-conf.json: |
+    {
+      "name": "cbr0",
+      "type": "flannel",
+      "delegate": {
+        "forceAddress": true,
+        "isDefaultGateway": true
+      }
+    }
+  net-conf.json: |
+    {
+      "Network": "100.64.0.0/16",
```

Comment:
Yeah totally, this was going to be my next step but ill update the PR to configure this based on the kops NonMasqueradeCIDR field